packages/redux/src/returns/reducer.ts

```diff
@@ -113,6 +132,16 @@ const pickupCapabilities = reducerFactory(
   actionTypes,
 );
 
+const pickupRescheduleRequests = reducerFactory(
+  [
+    'FETCH_RETURN_PICKUP_RESCHEDULE_REQUEST',
+    'FETCH_RETURN_PICKUP_RESCHEDULE_REQUESTS',
```

Comment:
Is this ok to mix the `FETCH_RETURN_PICKUP_RESCHEDULE_REQUEST` with `FETCH_RETURN_PICKUP_RESCHEDULE_REQUESTS` as they have a different payload?

Reply:
Actually, the payload is being ignored and because of that there is no selector for it. Is this correct? Shouldn't this be stored somewhere on the store and a selector be available to fetch it from there?

Reply:
Just leaving the note: We had a meet about this. This is something that needs to be decided the best way to be done (not having the results).